scripts/repair_wheel.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python3
+
+import argparse
+import shutil
+import subprocess
+import sys
+import tempfile
+from pathlib import Path
+
+
+def main():
+    if sys.platform.startswith("linux"):
+        os_ = "linux"
+    elif sys.platform.startswith("darwin"):
+        os_ = "macos"
+    elif sys.platform.startswith("win32"):
+        os_ = "windows"
+    else:
+        raise NotImplementedError(
+            "sys.platform '{}' is not supported yet.".format(sys.platform)
+        )
+
+    p = argparse.ArgumentParser(
+        description="Convert wheel to be independent of python implementation and ABI"
+    )
+    p.set_defaults(prog=Path(sys.argv[0]).name)
+    p.add_argument("WHEEL_FILE", help="Path to wheel file.")
+    p.add_argument(
+        "-w",
+        "--wheel-dir",
+        dest="WHEEL_DIR",
+        help=('Directory to store delocated wheels (default: "wheelhouse/")'),
+        default="wheelhouse/",
+    )
+
+    args = p.parse_args()
+
+    file = Path(args.WHEEL_FILE).resolve(strict=True)
+    wheelhouse = Path(args.WHEEL_DIR).resolve()
+    wheelhouse.mkdir(parents=True, exist_ok=True)
+
+    with tempfile.TemporaryDirectory() as tmpdir_:
+        tmpdir = Path(tmpdir_)
+        # use the platform specific repair tool first
+        if os_ == "linux":
+            subprocess.run(
+                ["auditwheel", "repair", "-w", str(tmpdir), str(file)], check=True
+            )
+        elif os_ == "macos":
+            subprocess.run(
+                [
+                    "delocate-wheel",
+                    "--require-archs",
+                    "arm64,x86_64",
```

Comment:
Not sure if we want this, this might be responsible for making a universal binary

Reply:
I'll uncomment it for another build.